### PR TITLE
Fix FadeOutDownTilesTest won't show last tile

### PR DESCRIPTION
**Pull request by xiaofeng11**

### CODE REVIEW

File: cocos/2d/CCActionTiledGrid.cpp [hunk 1]
```diff
@@ -572,12 +572,12 @@ FadeOutDownTiles* FadeOutDownTiles::clone() const
...
-    if (pos.height == 0)
-    {
-        return 1.0f;
-    }
+    //if (pos.height == 0)
+    //{
+    //    return 1.0f;
+    //}
 
-    return powf(n.y / pos.height, 6);
```
Comment (pandamicro) [inline cocos/2d/CCActionTiledGrid.cpp:580]: Could pos.height be negative here ?

Comment (xiaofeng11) [inline cocos/2d/CCActionTiledGrid.cpp:580]: time value will be always <= 1.0, so pos.height won't be negative

**Comment by pandamicro** [comment]
Review ok

File: cocos/2d/CCActionTiledGrid.cpp [hunk 2]
```diff
@@ -572,12 +572,12 @@ FadeOutDownTiles* FadeOutDownTiles::clone() const
...
 {
     Vec2 n = Vec2((float)_gridSize.width, (float)_gridSize.height) * (1.0f - time);
-    if (pos.height == 0)
-    {
-        return 1.0f;
-    }
+    //if (pos.height == 0)
+    //{
+    //    return 1.0f;
+    //}
```
Comment (pandamicro) [inline cocos/2d/CCActionTiledGrid.cpp]: Please remove these lines

Comment (xiaofeng11) [inline cocos/2d/CCActionTiledGrid.cpp]: removed

**Pull request merged**